src/ONYXKEYS.js

```diff
@@ -68,4 +68,7 @@ export default {
         REPORT_USER_IS_TYPING: 'reportUserIsTyping_',
         REPORT_IOUS: 'reportIOUs_',
     },
+
+    // Indicates which locale should be used
+    PREFERRED_LOCALE: 'en',
```

Comment:
@iwiznia Just confirming this is the correct place for this. Do we already have an action to change this key?

Reply:
Yes, this is the correct place and no, we don't yet have an action to change it